lambdas/service/app.py

```diff
@@ -43,6 +44,18 @@
 configure_app_logging(app, log)
 
 
+api_info = {
```

Comment:
Can we consolidate the terminology a bit? If it is `spec` for the decorator and keyword argument, it should be `spec` here as well. OTOH, `spec` is a bit too generic to be used at the module scope. That's why I'd prefer `openapi` for everything.

Reply:
What about something like `top_level_spec` or `global_spec`? `openapi` just parses kind of funny for me.

Reply:
`top_level_spec` is long and awkward. Both "global" and "toplevel" are implied because the variable is defined at the module scope. I like `openapi` because it is both short and unique. Can you explain what you mean by "parses funny"?